javascript/javascript1/week3/travel_info.js

```diff
@@ -0,0 +1,15 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function calculateTime(travelInformation){
+     
+      
+    if(travelInformation.speed>0 && travelInformation.destinationDistance>=0){
+       
+        return Math.floor(travelInformation.destinationDistance*60/travelInformation.speed);//calculates travel time in munites
+      }
```

Comment:
and what happens if the condition is not satisfied? Your function should still return something